src/App.css

```diff
@@ -90,7 +90,14 @@
 
 .main-max-width {
   max-width: var(--main-max-width);
-  margin: auto;
+  height: auto !important;
```

Comment:
I don't think that you need this line unless there is an edge case that I am missing.